ReactiveXComponent/WebSocket/WebSocketMessageHelper.cs

```diff
@@ -135,13 +134,13 @@ public static WebSocketMessage DeserializeRequest(string request)
                 int firstDotIndex = firstPartBeforeJson.IndexOf('.');
 
                 // Assume it is an input on a custom type of topic if it doesn't have the format  topic = {key}.*.*.* (e.g. output.1_0.engine1.OrderService.CreationFacade)
-                key = firstDotIndex >= 0 ? firstPartBeforeJson.Substring(0, firstDotIndex) : WebSocketCommand.Input;
-                topic = tokensBeforeJson[0];
+                key = firstDotIndex > 0 ? firstPartBeforeJson.Substring(0, firstDotIndex) : WebSocketCommand.Input;

```

Comment:
The whole method was taken as is from XComponent Core code (WebSocket bridge). Yes it can later be refactored to reduce complexity and potential bugs.